Access API/AccessAPI/BLL/SearchBLL.cs

```diff
@@ -1,23 +1,26 @@
 ﻿using System;
+using System.Collections.Generic;
 using Access_API.DAL;
 
 namespace Access_API.BLL
 {
     public class SearchBLL
     {
-        public string SearchBll(string words, int? sourceId, string? author, int? categoryId,
-            DateTime? beforeDate, DateTime? afterDate)
+        public string SearchBll(string words, List<long>? sourceIds, List<string>? authors, List<int>? categoryId,
+            DateTime? beforeDate, DateTime? afterDate, int? limit, int? offset)
         {
             string url = Urls.SearchUrl + $"/search?words={words}";
-            if (sourceId is not null) url += $"&sourceId={sourceId}";
-            if (author is not null) url += $"&author={author}";
+            if (sourceIds is not null) url += $"&sourceId={sourceIds}";
+            if (authors is not null) url += $"&author={authors}";
             if (categoryId is not null) url += $"&categoryId={categoryId}";
```

Comment:
```suggestion
            foreach(long sourceId in sourceIds) url += $"&sourceIds={sourceId}";
            foreach(string author in authors) url += $"&authors={author}";
            foreach(int categoryId in categoryIds) url += $"&categoryIds={categoryId}";
```
Skrevet om til at matche ASP.NET Core's håndtering af lists i query parameters!